### ISSUE TITLE
can not run in gui and The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by Dragonkingpan** [Lakonik/SSDNeRF#20]
I want to report the following error when using the GUI interface to display.
```
python demo/ssdnerf_gui.py configs/paper_cfgs/ssdnerf_cars_uncond.py work_dirs/ssdnerf_cars_uncond/save/0000.pth --fp16
2023-08-14 17:52:19,420 - mmgen - INFO - Apply 'timestep_weight' rescale_mode for loss_ddpm_mse. Please make sure the passed weight can be updated by external functions.
load checkpoint from local path: work_dirs/ssdnerf_cars_uncond/save/0000.pth
The model and loaded state dict do not match exactly
unexpected key in source state_dict: scene_name, param
```
 In addition, I also wanted to know if I could export a model with a map, the surface of the exported STL model file was a bit rough, and whether there was a way to improve the accuracy of the model.

**Comment by Dragonkingpan** [comment]
![1692007466147](https://github.com/Lakonik/SSDNeRF/assets/33677417/a7bf1254-9f85-41c6-a57c-63da3abea135)

**Comment by Lakonik** [comment]
Hi! In the command line you need to use the model checkpoints instead of saved scenes, e.g.

```
python demo/ssdnerf_gui.py configs/paper_cfgs/ssdnerf_cars_uncond.py work_dirs/ssdnerf_cars_uncond/ckpt/ssdnerf_cars_uncond/latest.pth --fp16
```

Then you can load the saved scenes in the GUI for visualization.

The triplane representation is known to produce stair-step artifacts, so the normals of the mesh may not look good. You may try some mesh processing tools to smoothen the results. Also, the rgb values can also be saved as vertex colors (we may add exporting colored mesh in a future release).
